templates/zerver/help/mute-a-stream.md

```diff
@@ -0,0 +1,23 @@
+### Mute a stream
+
+Muting streams to reduce notification clutter is easy. Simply hover your mouse
+over the stream in the streams list on the left of the chat, press the down
+chevron button
+
+![streams arrow](static/images/help/streams-arrow.png)
+
+to bring up the stream popover menu and press **Mute the stream**. The muted
+stream should be greyed out in the streams list.
+
+![stream popover](/static/images/help/stream-left-sidebar-color.png)
+
+### Unmute a stream
+
+To unmute a stream, hover your mouse over the greyed stream in the streams
+list, press the down chevron button
+
+![muted stream](/static/images/help/muted_stream.png)
```

Comment:
Our docs usually don't have pictures that appear in the middle of sentences. Showing a picture of the chevron appearing is a bit unnecessary, so you can use the inline chevron image again by using something like `click the chevron (![chevron](static/images/help/chevron.png)) icon and press **Unmute the stream**`